src/metabase/util/ssh.clj

```diff
@@ -6,12 +6,20 @@
 
 (def ^:private default-ssh-timeout 30000)
 
+(defn ssh-add-key
+  "Adds SSH private key to Jsch class."
+  [jsch key-file-name]
+  (log/debug "Adding ssh key:" key-file-name)
+  (when-not (string/blank? key-file-name) (doto ^com.jcraft.jsch.JSch jsch (.addIdentity key-file-name)))
```

Comment:
I have tested in my local with `str/blank?` patched, and this pull request worked on my machine. I think metabase should setup some kind of lint tests like eastwood.